HW04/ivan_klimov/lib/notification.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Notification
+  def check_for_active_notifications(person)
```

Comment:
we usually use the following names 
```suggestion
  def active_notifications_present?(person)
```